transforms/he_class_icons.py

```diff
@@ -0,0 +1,27 @@
+{
+    'data': [
+        ('vm:HE/IntangibleAsset', 'Intangible Asset'),
+        ('vm:HE/TangibleAsset', 'Tangible Asset'),
+        ('http://webprotege.stanford.edu/RCnRceKsHZf8Gt9UvDjM6We',
+            'Delivery partner'),
```

Comment:
Is this a typo or is the `partner` in the icon filename also uncapitalised?

Reply:
Yeah, I didn't normalise the naming, it's just what @Barbara-OK saved things as.